pkg/api/types/v1/component.go

```diff
@@ -90,15 +99,15 @@ type TestOwnership struct {
 	JIRAComponent string `bigquery:"jira_component"`
 
 	// JIRAComponentID specifies the ID of the JIRA component above.
-	JIRAComponentID bigquery.NullInt64 `bigquery:"jira_component_id"`
+	JIRAComponentID *big.Rat `bigquery:"jira_component_id"`
```

Comment:
What's the motivation to change this to a `Rat`? Rational allows fractional numbers (e.g. 1/3) which doesn't make sense for JIRAComponentID.

Reply:
I was getting an error when listing the table. Numeric does not work with bigquery.NullInt64. Let me research a little more to see if there are other options for Numeric.

Reply:
I see, disregard.  You can leave it as Rat.

The schema should've been `bigquery.IntegerFieldType` when I created it.  Numeric allows decimal values so Rat is fine.

https://github.com/openshift-eng/ci-test-mapping/blob/88bc5a654e0c98eec98129578a756934ce8d0f4c/pkg/api/types/v1/component.go#L134-L137

Reply:
I think this means Rat is the only option: https://github.com/openshift-eng/ci-test-mapping/blob/88bc5a654e0c98eec98129578a756934ce8d0f4c/vendor/cloud.google.com/go/bigquery/value.go#L748